recipes/pandasdmx/meta.yaml

```diff
@@ -0,0 +1,45 @@
+    {% set name = "pandaSDMX" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 799d53f8c8a49321fc92d8bf0f6bf7101b0ffe14c7f2768aef33776b7eae48f6
+
+build:
+  number: 0
+  skip: true  # [py<37]  
```

Comment:
I think you can remove this `skip` statement, add `noarch: python` (https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) and change `python` to `python >=3.7` below under the `host` and `run` requirements sections.

Reply:
Thanks. Works. Anything else to do? Next steps?